repos/system_upgrade/el7toel8/libraries/testutils.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, arch=architecture.ARCH_X86_64, envars=None, kernel='3.10.0-95
         self.configuration = namedtuple(
             'configuration', ['architecture', 'kernel', 'leapp_env_vars', 'os_release', 'version']
         )(arch, kernel, envarsList, release, version)
-        self._msgs = msgs
+        self._msgs = msgs if msgs else []
```

Comment:
```suggestion
        self._msgs = msgs or []
```

Reply:
alternatively you could probably set msgs=() in the constructor

Reply:
Thx!